BUILD

```diff
@@ -2035,21 +2037,21 @@ grpc_cc_library(
     ],
 )
 
-grpc_cc_library(
-    name = "s2a_util",
-    srcs = [
-        "src/core/lib/security/credentials/s2a/grpc_s2a_credentials_options.cc",
-    ],
-    hdrs = [
-        "src/core/lib/security/credentials/s2a/grpc_s2a_credentials_options.h",
-    ],
-    language = "c++",
-    public_hdrs = GRPC_SECURE_PUBLIC_HDRS,
-    deps = [
-        "gpr",
-        "grpc_base",
-    ],
-)
+#grpc_cc_library(
```

Comment:
Maybe remove this rule, instead of commenting it out?

Reply:
Thanks! I forgot to delete it.